imports/startup/server/register-api.js

```diff
@@ -38,3 +38,6 @@ import '/imports/api/emailTemplates/methods';
 import '/imports/api/engage/server/publications';
 import '/imports/api/engage/server/methods';
 import '/imports/api/engage/server/cronJobs';
+
+import '/imports/api/knowledgebase/server/publications';
+import '/imports/api/knowledgebase/server/methods';
```

Comment:
Can KB methods be defined on both of client and server? https://guide.meteor.com/methods.html#defining-and-calling

Reply:
i understand the theory, but how do i implement it?

Reply:
done